source/includes/reference/_webhook_events.md

```diff
@@ -87,155 +87,166 @@ Test messages can be sent to verify callback URLs when subscriptions are being s
 
 ## Event payload
 
+All event notification payloads have the same high-level structure.
+Top-level properties are common to all events.
+The `data` property is an object that can contain various properties.
+The exact properties that the `data` object contains depends on the event type and schema version of the event.  
+
+
 > Basic event payload:
 
 ```json
 {
   "data": {},
-  "subscription_id": "db463333-3333-3333-3333-185d1ee81585",
-  "event_type": "...",
+  "subscription_id": "01234567-89ab-cdef-0123-456789abcdef",
+  "event_type": "event#type",
   "schema_version": "2.0.0",
-  "sent_at": "2019-10-14T12:43:37Z"
+  "sent_at": "2020-01-01T12:34:56Z"
 }
 ```
 
-All events share same high-level payload structure.
-Depending on the `event_type`, the only object that differs between events is `data`.
+### Common properties
 
-Field           | Description                                     | Format
----------       | -------                                         | -----------
-data            | Object with information about updated resource  | Object
-subscription_id | ID of the subscription that triggered the event | UUID
-event_type      | Type of event, which defines `data` object      | Integer
-schema_version  | The event representation semantic version       | String
-sent_at         | Timestamp when update happened                  | Timestamp
+Field           | Description                                                                   | Format
+---------       | -------                                                                       | -----------
+data            | Event type- and schema version-specific details                               | Object
+subscription_id | ID of the webhook subscription that triggered the event notification          | UUID
+event_type      | Event type (what event happened in our system)                                | String
+schema_version  | Schema version (what notification structure is being used to model the event) | String
+sent_at         | When the event notification was sent from our system                          | Datetime
 
 
 ## Transfer status change event
 
-> Example event:
+Event type: `transfers#state-change`
+
+This event will be triggered every time a transfer's status is updated. Each event contains a timestamp.
+As we do not guarantee the order of events, please use `data.occurred_at` to reconcile the order. 
+
+If you would like to subscribe to transfer state change events, please use `transfers#state-change`
+when creating your subscription.
+
+
+> Example v2.0.0 `transfers#state-change` event:
```

Comment:
The examples are event type AND schema version specific, so I made that more obvious.